js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -617,53 +619,6 @@ const MiddlePanel = ({
     );
   }
 
-  if (!evaluationData) {
-    if (selectedEvaluationId) {
-      return (
-        <Box flex={{direction: 'column', grow: 1}}>
-          <Box flex={{direction: 'row', justifyContent: 'center'}} padding={{vertical: 24}}>
-            <NonIdealState
-              icon="auto_materialize_policy"
-              title="Evaluation not found"
-              description={
-                <>
-                  No evaluation with ID <Mono>{selectedEvaluationId}</Mono> found
```

Comment:
Now it goes to no selection if the evaluation id isn't found

Reply:
what if its an old evaluation that was deleted and they have a link to it saved somehow? Maybe we should indicate that it was deleted? (once we start pruning)

Reply:
Yeah we could figure something out